Files/Scripts/Options/Majora's Mask.psm1

```diff
@@ -1256,13 +1256,24 @@ function ByteTextOptions() {
-        SetMessage -ID "004C" -Text "<W> Buttons" -Replace "<W>"; SetMessage -ID "0059" -Text ", then press that <N><C><C Button> <W>Button to use it." -Replace "<W>."; SetMessage -ID "1700" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press" # Buttons -> C Icon Only	    
-        
-        SetMessage -ID "001A" -Text "Press" -Replace "Use"; SetMessage -ID "0022" -All; SetMessage -ID "0028"; SetMessage -ID "0029"; SetMessage -ID "002A"; SetMessage -ID "002E"; SetMessage -ID "0034"; SetMessage -ID "0036"; SetMessage -ID "003A"; SetMessage -ID "0041"; SetMessage -ID "0043"; SetMessage -ID "1701" # Press -> Use
-        SetMessage -ID "1706";                              SetMessage -ID "1709";      SetMessage -ID "170E"; SetMessage -ID "170F"; SetMessage -ID "1717"; SetMessage -ID "171B"; SetMessage -ID "172E"; SetMessage -ID "0079"; SetMessage -ID "007A"
+	SetMessage -ID "004C" -Text "four <C><C Button><W> Buttons to play it. Press<N><B Button> to stop." -Replace "four <C><C Button><W> directions to play it.<N>Press <B Button> to stop."
+        SetMessage -ID "1700" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press"
+	
+        SetMessage -ID "001A" -Text "Press" -Replace "Use";      SetMessage -ID "0022" -All; SetMessage -ID "0028"; SetMessage -ID "0029"; SetMessage -ID "002A"; SetMessage -ID "002E"; SetMessage -ID "0034"; SetMessage -ID "0036"
+        SetMessage -ID "003A"; SetMessage -ID "0041"; SetMessage -ID "0078"; SetMessage -ID "1701"; SetMessage -ID "1706"; SetMessage -ID "1709"; SetMessage -ID "170E"; SetMessage -ID "170F"; SetMessage -ID "1717"
+	SetMessage -ID "171B"; SetMessage -ID "172E"
+
+        SetMessage -ID "0014" -Text "press" -Replace "use"; SetMessage -ID "0015"; SetMessage -ID "0016"; SetMessage -ID "0017"; SetMessage -ID "0018"; SetMessage -ID "0019"; SetMessage -ID "0035"; SetMessage -ID "0200"; SetMessage -ID "09E3"
+	SetMessage -ID "1F4F"
+
+        SetMessage -ID "0078"; -Text "the body of a Deku. Press <C><C Button>" -Replace "the body of a Deku. Use <C><C Button>"
+	SetMessage -ID "0079"; -Text "the body of a Goron. Press <C><C Button>" -Replace "the body of a Goron. Use <C><C Button>"
+        SetMessage -ID "007A"; -Text "the body of a Zora. Press <C><C Button>" -Replace "the body of a Zora. Use <C><C Button>"
+
+        SetMessage -ID "0043" -Text "Press <C><C Button> <W>to look through it" -Replace "Use <C><C Button> <W>to look through it"
+	SetMessage -ID "0059" -Text "then press that <N><C><C Button> <W>Button to use it." -Replace "then use <N><C><C Button><W>."
+
+        if (IsChecked $Redux.Features.OcarinaIcons) { SetMessage -ID "1726" -Text "<C Button><N>Buttons. Press" -Replace "<C Button>.<N>Press"; SetMessage -ID "171C"; SetMessage -ID "1727" }
```

Comment:
Is this option check required? Shouldn't it also apply for the original Ocarina icon when transformed as well?

Reply:
> Is this option check required? Shouldn't it also apply for the original Ocarina icon when transformed as well?

It's for only if the **Ocarina Icons** is enabled. Otherwise, it will just patch the vanilla **Ocarina of Time** one.

Reply:
I see... Right... I forgot lol.

Reply:
> I see... Right... I forgot lol.
> 
> But isn't the option a ComboBox rather than a Checkbox? IsChecked won't work on a ComboBox option.

Yeah. I fixed this. It's now **Index** sensitive.